datamodel.customer-survey.xml

```diff
@@ -1891,6 +1892,8 @@ EOF
 			$sSubject = $this->Get('email_subject');
 		}
 		
+		$sSubject = $this->ApplyParamsToContent($sSubject, $oContact, null);
```

Comment:
I honestly can imagine very few scenarios where someone wants to put the URL in the subject; but why would we make it impossible?

Reply:
I was thinking this as well, but there is a small detail, the URL parameter is set to `null` here.

Reply:
The intent is not very clear here indeed : I passed null mainly because we never want to append the link on the subject.
So maybe we should add a boolean parameter for this behavior ? Something like `bShouldAppendLinkIfNoUrlPlaceholder` ?

Reply:
It's a bit long, no? `bAutoAppendLink` should be enough, you still have the php doc that can explain more if needed.

Reply:
Done in 770f2437